gridstatus/pjm.py

```diff
@@ -2105,3 +2105,55 @@ def _parse_load_metered_hourly(self, df):
         ]
 
         return df.sort_values("Interval Start").reset_index(drop=True)
+
+    @support_date_range(frequency=None)
+    def get_forecasted_generation_outages(self, date, end=None, verbose=False):
+        """
+        Retrieves the forecasted generation outages for the next 90 days from:
+
+        https://dataminer2.pjm.com/feed/frcstd_gen_outages/definition
+        """
+        if date == "latest":
+            date = "today"
+
+        df = self._get_pjm_json(
+            "frcstd_gen_outages",
+            start=date,
+            params={
+                "fields": "forecast_execution_date_ept,forecast_date,forecast_gen_outage_mw_rto,forecast_gen_outage_mw_west,forecast_gen_outage_mw_other",  # noqa: E501
+            },
+            end=end,
+            filter_timestamp_name="forecast_execution_date",
+            interval_duration_min=1440,
+            verbose=verbose,
+        )
+
+        return self._parse_load_metered_hourly(df)
+
+    def _parse_load_metered_hourly(self, df):
```

Comment:
Nit: method name looks like it needs to be updated.

Reply:
Yep - copypasta. Thanks.